pithos/pithos.py

```diff
@@ -900,20 +900,14 @@ def on_gst_volume(self, player, volumespec):
         GLib.idle_add(self.set_volume_cb, vol)
 
     def on_gst_source(self, player, params):
-        """ Setup httpsoupsrc to match Pithos proxy settings
-            and give the user time to reconnect in case of a
-            network disconnect.
-        """
-        soup = player.props.source.props
-        if hasattr(soup, 'timeout'):
-            soup.timeout = 600
-        if hasattr(soup, 'proxy'):
-            proxy = self.get_proxy()
-            if proxy:
-                scheme, user, password, hostport = parse_proxy(proxy)
-                soup.proxy = hostport
-                soup.proxy_id = user
-                soup.proxy_pw = password
+        """ Setup httpsoupsrc to match Pithos proxy settings """
+         soup = player.props.source.props
+         proxy = self.get_proxy()
+         if proxy and hasattr(soup, 'proxy'):
+             scheme, user, password, hostport = parse_proxy(proxy)
+             soup.proxy = hostport
+             soup.proxy_id = user
+             soup.proxy_pw = password
```

Comment:
__[flake8]__
* [E111] indentation is not a multiple of four

<!-- Comment by [SideCI](https://sideci.com) -->